Makefile

```diff
@@ -16,21 +16,19 @@ help:
 test:
 	$(gradle) check
 
+# extract the currently targeted version of LightGBM
+lightGbmVersion := $(shell grep lightGbmVersion gradle.properties | sed 's/.*lightGbmVersion=//')
```

Comment:
Nice that you've connect this to `gradle.properties`. `build-lightgbm.sh` uses it to checkout a tag from the microsoft/LightGBM repo, which are prefixed with `v`, eg: `v2.2.2`, `v2.3.0`. 

Suggest prefixing the `v` here, eg:
```suggest
lightGbmVersion := v$(shell grep lightGbmVersion gradle.properties | sed 's/.*lightGbmVersion=//')
```